src/modules/TodoItem.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable react/prop-types */
+
+import { useState } from 'react';
+import styles from '../styles/TodoItem.module.css';
+
+const TodoItem = ({
+  itemProp, handleChange, delTodo, setUpdate,
+}) => {
+  const [editing, setEditing] = useState(false);
+
+  const viewMode = {};
+  const editMode = {};
+  if (editing) {
+    viewMode.display = 'none';
+  } else {
+    editMode.display = 'none';
+  }
+  const handleEditing = () => {
+    setEditing(true);
+  };
+
+  const handleUpdatedDone = (event) => {
+    if (event.key === 'Enter') {
+      setEditing(false);
+    }
+  };
+  return (
+    <li className={styles.item}>
+      <div className={styles.content} style={viewMode}>
+        <input
+          type="checkbox"
+          checked={itemProp.completed}
+          onChange={() => handleChange(itemProp.id)}
+        />
```

Comment:
- [ Optional] I suggest that you add style so as to mark the activity as complete or not. Currently the checkbox is only checking but the item is remaining the same
 
![Todo screen re](https://user-images.githubusercontent.com/98940605/235591465-b083d080-0525-4600-b001-8a1160cee404.jpg)